app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -0,0 +1,233 @@
+h1 {
```

Comment:
wrap your styles into when you need to add styling for all screen sizes:
`& when (@media-common = true) {}`

Reply:
done